pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -3383,6 +3383,95 @@ func Test_server_MultiInsert(t *testing.T) {
+				- case 1.5: Success to MultiInsert 0 vector (vector type is uint8)
+				- case 1.6: Success to MultiInsert 0 vector (vector type is float32)
+				- case 2.1: Fail to MultiInsert 1 vector with different dimension (vector type is uint8)
+				- case 2.2: Fail to MultiInsert 1 vector with different dimension (vector type is float32)
+				- case 3.1: Fail to MultiInsert 100 vector with 1 vector with different dimension (vector type is uint8)
+				- case 3.2: Fail to MultiInsert 100 vector with 1 vector with different dimension (vector type is float32)
+				- case 3.3: Fail to MultiInsert 100 vector with 50 vector with different dimension (vector type is uint8)
+				- case 3.4: Fail to MultiInsert 100 vector with 50 vector with different dimension (vector type is float32)
+				- case 3.5: Fail to MultiInsert 100 vector with all vector with different dimension (vector type is uint8)
+				- case 3.6: Fail to MultiInsert 100 vector with all vector with different dimension (vector type is float32)
+
+		- Boundary Value Testing
+			- uint8, float32 (with 100 insert request in a single MultiInsert request)
+				- case 1.1: Success to MultiInsert with 0 value vector (vector type is uint8)
+				- case 1.2: Success to MultiInsert with 0 value vector (vector type is float32)
+				- case 2.1: Success to MultiInsert with min value vector (vector type is uint8)
+				- case 2.2: Success to MultiInsert with min value vector (vector type is float32)
+				- case 3.1: Success to MultiInsert with max value vector (vector type is uint8)
+				- case 3.2: Success to MultiInsert with max value vector (vector type is float32)
+				- case 4.1: Fail to MultiInsert with 1 request with empty UUID (vector type is uint8)
+				- case 4.2: Fail to MultiInsert with 1 request with empty UUID (vector type is float32)
+				- case 4.3: Fail to MultiInsert with 50 request with empty UUID (vector type is uint8)
+				- case 4.4: Fail to MultiInsert with 50 request with empty UUID (vector type is float32)
+				- case 4.5: Fail to MultiInsert with all request with empty UUID (vector type is uint8)
+				- case 4.6: Fail to MultiInsert with all request with empty UUID (vector type is float32)
+				- case 5.1: Fail to MultiInsert with 1 vector with maximum dimension (vector type is uint8)
+				- case 5.2: Fail to MultiInsert with 1 vector with maximum dimension (vector type is float32)
+				- case 5.3: Fail to MultiInsert with 50 vector with maximum dimension (vector type is uint8)
+				- case 5.4: Fail to MultiInsert with 50 vector with maximum dimension (vector type is float32)
+				- case 5.5: Fail to MultiInsert with all vector with maximum dimension (vector type is uint8)
+				- case 5.6: Fail to MultiInsert with all vector with maximum dimension (vector type is float32)
+
+			- float32 (with 100 insert request in a single MultiInsert request)
+				- case 6: Success to MultiInsert with NaN value (vector type is float32)
```

Comment:
How about +-Inf, -0 case in float32?
```suggestion
				- case 6.1: Success to MultiInsert with NaN value (vector type is float32)
				- case 6.2: Success to MultiInsert with +Inf value (vector type is float32)
				- case 6.3: Success to MultiInsert with -Inf value (vector type is float32)
				- case 6.4: Success to MultiInsert with -0 value (vector type is float32)
```